prowler/providers/azure/azure_provider.py

```diff
@@ -32,6 +35,9 @@ def __init__(
             subscription_ids,
         )
 
+    def __set_region_config__(self, region):
```

Comment:
Why this function is called `__set_region_config__` but then is calling `get_regions_config`, maybe you can call directly `get_regions_config`

Reply:
Done